demo/checkbox-basic-demos.html

```diff
@@ -45,6 +45,32 @@ <h3>Disabled state</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Helper text</h3>
+    <p>Use the <code>helper-text</code> attribute or add content to the <code>helper</code> slot to set helper content.</p>
+    <vaadin-demo-snippet id="text-field-basic-demos-helper-text">
```

Comment:
Mentions text-field in the id. I think the component doesn't need to be mentioned in the demo id. Just "helper-text" should be enough for a demo id

Reply:
Done.